index.js

```diff
@@ -16,7 +15,8 @@ module.exports = options => {
 
   const load = (config) => {
     config.routes.forEach((route) => {
-      app.use(router(route, config));
+      config.route = route;
```

Comment:
there is no `config.route` before this. I added this to simplify the arguments passed to router